### PR TITLE
Check if RSVP or Tickets are available

### PR DESCRIPTION
**Pull request by sumbria**

### CODE REVIEW

File: includes/public/class-cep-summit-public.php [hunk 1]
```diff
@@ -86,8 +86,15 @@ public function on_init()
     }
 
     public function remove_original_ticket_and_rspv_forms_and_add_ours() {
-        remove_action( 'tribe_events_single_event_after_the_meta', array( Tribe__Tickets__RSVP::get_instance(), 'maybe_add_front_end_tickets_form' ), 5 );
-        remove_action( 'tribe_events_single_event_before_the_meta', array( Tribe__Tickets__Commerce__PayPal__Main::get_instance(), 'maybe_add_front_end_tickets_form' ), 5 );
+        foreach( [
+            'tribe_events_single_event_before_the_content',
+            'tribe_events_single_event_after_the_content',
+            'tribe_events_single_event_before_the_meta',
+            'tribe_events_single_event_after_the_meta',
```
Comment (brthompsonSirsi) [inline includes/public/class-cep-summit-public.php]: should this have a comma if it's the end of the list?

File: tribe-events/modules/meta.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ * @version 4.6.10
+ *
+ * @package TribeEventsCalendar
+ */
+
+do_action( 'tribe_events_single_meta_before' );
+
+// Check for skeleton mode (no outer wrappers per section)
+$not_skeleton = ! apply_filters( 'tribe_events_single_event_the_meta_skeleton', false, get_the_ID() );
```
Comment (brthompsonSirsi) [inline tribe-events/modules/meta.php]: is there supposed to be a space between ! and apply_filters?

Comment (sumbria) [reply tribe-events/modules/meta.php]: Yes it's auto formatting. Whit spaces will be ignored during compile time.

File: includes/public/class-cep-summit-public.php (hunk 1 above)
Comment (sumbria) [reply includes/public/class-cep-summit-public.php]: Comma is optional here.

**Pull request merged**